docs/e2e-tests/README.md

```diff
@@ -0,0 +1,91 @@
+
+- **Node.js** (minimum version 18)
+- An instance of the application to run the tests against
+- [Playwright browsers installed](#install-playwright-browsers)
+
+#### Environment Variables
+
+Certain environment variables need to be set up, depending on what you intend to run. The most convenient way is to export them from the CLI or add them in your `.bash_profile` or `.zshrc`. Alternatively, they can be passed to Playwright via the `--env` flag:
+
+```bash
+# BASE_URL (The URL to the main page of the application) is mandatory to run all the E2E tests.
+VAR_NAME=variable_value npx playwright test
+```
+
+The currently supported environment variables are:
+
+| Variable Name            | Description                                                | Required for Tests                      |
+| ------------------------ | ---------------------------------------------------------- | --------------------------------------- |
+| `BASE_URL`               | The URL to the main page of the application                | All tests                               |
+| `GH_USER_ID`             | Your GitHub username, required for logging in using GitHub | Tests involving GitHub authentication   |
+| `GH_USER_PASS`           | Your GitHub password                                       | Tests involving GitHub authentication   |
+| `GH_2FA_SECRET`          | GitHub 2FA secret used to generate a 2FA OTP for login     | Tests involving GitHub authentication   |
+| `GH_USER_TOKEN`          | A classic GitHub token used to make API calls to GitHub    | Tests involving GitHub API interactions |
+| `KEYCLOAK_BASE_URL`      | Keycloak base URL                                          | Tests involving Keycloak authentication |
+| `KEYCLOAK_REALM`         | Keycloak realm                                             | Tests involving Keycloak authentication |
+| `KEYCLOAK_CLIENT_ID`     | Keycloak client ID                                         | Tests involving Keycloak authentication |
+| `KEYCLOAK_CLIENT_SECRET` | Keycloak client secret                                     | Tests involving Keycloak authentication |
+
+#### Running the Tests
+
+The Playwright command line supports many options; see them [here](https://playwright.dev/docs/test-cli). Flags like `--ui` or `--headed` are very useful when debugging. You can also specify a specific test to run:
+
+```bash
+npx playwright test e2e-tests/playwright/e2e/your-test-file.spec.ts
+```
+
+Our project contains multiple test suites for different environments and configurations. Some useful scripts to run the tests:
+
+```bash
+yarn showcase                       # Runs the showcase test suite
+yarn showcase-rbac                  # Runs the showcase RBAC test suite
+yarn showcase-1-2-x                 # Runs the showcase 1.2.x test suite
+yarn showcase-rbac-1-2-x            # Runs the showcase RBAC 1.2.x test suite
+```
+
+For more information on how the tests run in the CI, see [this document](CI.md)
```

Comment:
@nilgaar Is this `CI.md` document part of this PR too?  I was just wondering if there's a section covering how to interpret the test results, and how to figure out which test failed and where to look.

Reply:
hi @gashcrumb ! The CI.md is in this other one right here: https://github.com/janus-idp/backstage-showcase/pull/1802.
I'll come back to the CI.md today, as I had a problem with a diagram when see in dark mode. On the meantime I thought we can reference to it already, as it shall be merged soon